src/SupplyChain/SupplyChain/ViewModels/MainViewModel.cs

```diff
@@ -40,5 +43,37 @@ public MainViewModel()
                 }
             }
         });
+
+        public ICommand AddCommand => new CommandHandler(() =>
+        {
+            using (var connection = new SqlConnection(_connectionString))
+            {
+                var sql = @"INSERT INTO [User] ([FirstName], [LastName], [UpdatedDate]) VALUES (@FirstName, @LastName,GETUTCDATE())";
+
+                string text1 = mainWin.textBox1.Text;
+                string text2 = mainWin.textBox2.Text;
+                UserViewModel user = new UserViewModel();
+                user.FirstName = text1;
+                user.LastName = text2;
+
+                connection.Execute(sql, user);
```

Comment:
Use dynamic object like in the example I sent you https://dotnetfiddle.net/Z1iRIQ instead of UserViewModel. It has a different purpose of usage.